From fd38be8256c9865cdfc2903328c41de4d4616dac Mon Sep 17 00:00:00 2001
From: Chen-Yu Tsai <wenst@chromium.org>
Date: Fri, 14 Jan 2022 04:33:46 +0000
Subject: [PATCH 01/17] media/gpu/v4l2: VD: Add sequenced_task_runner_handle.h
 header

In commit e4bb5ae4828be ("media/gpu/chromeos: make sure VideoDecoderMixins
respect Decode()"), usage of base::SequencedTaskRunnerHandle was added,
but its defining header was not included.

This might have worked with ChromeOS builds through some indirect
inclusion of said header, but it doesn't work when building for standard
Linux.

Include said header in v4l2_video_decoder.cc where SequencedTaskRunnerHandle
is used.

BUG=b:192087509
TEST=Build for standard Linux

Change-Id: I614788d0a18a3a7f05f99730902d8ca11b1bc820
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3380424
Reviewed-by: Miguel Casas-Sanchez <mcasas@chromium.org>
Commit-Queue: Chen-Yu Tsai <wenst@chromium.org>
Cr-Commit-Position: refs/heads/main@{#959037}
---
 media/gpu/v4l2/v4l2_video_decoder.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/media/gpu/v4l2/v4l2_video_decoder.cc b/media/gpu/v4l2/v4l2_video_decoder.cc
index c75a9148..7534052f 100644
--- a/media/gpu/v4l2/v4l2_video_decoder.cc
+++ b/media/gpu/v4l2/v4l2_video_decoder.cc
@@ -11,6 +11,7 @@
 #include "base/logging.h"
 #include "base/memory/ptr_util.h"
 #include "base/task/post_task.h"
+#include "base/threading/sequenced_task_runner_handle.h"
 #include "base/trace_event/trace_event.h"
 #include "media/base/limits.h"
 #include "media/base/media_log.h"
-- 
2.17.1

